src/jobs/go-architect-legacy.yaml

```diff
@@ -0,0 +1,12 @@
+description: |
+  This job runs the given command for the current project from a container
+  using the `latest` architect image and ultimately calling `architect <command>`.
+machine: true
```

Comment:
I wonder if we can use architect executor here and then not use docker in the command.

Reply:
I think I had an issue when I did that at first, but I will confirm.

Reply:
Now I remember. Here's what happened https://circleci.com/gh/giantswarm/apiextensions/1369 The CircleCI runner launches the build in the `architect` container. Then the `architect` binary calls `docker` commands so we have to support docker-in-docker by passing in the docker socket to the `architect` container and I could only get that to work by using `machine: true`.

Reply:
You have `setup_remote_docker` settings and you can run docker from docker executor. It has limitation of containers not being able to communicate via network but that should be fine for architect. But I'm not big on that. I'm big on removing this job from the orb ASAP (and it's not merged yet :D).

Reply:
I can't figure out an easy way to get the repo onto the remote docker server so I think I'm going to go with the `machine`-based approach for now. As you say, this job is intended to go away in the near future, and I don't want to spend more time on it. And it works as it is.